manifests/authentication/certificate.pp

```diff
@@ -96,6 +96,16 @@
         require => Package[$lightblue::eap::package_name],
     }
 
+    # create ldap-healthcheck-config.json file required to initialize RolesProvider
+    file { "${directory}/ldap-healthcheck-config.json":
```

Comment:
Where is ${directory} coming from?  This should ideally get landed in the lightblue JBoss module directory (so it is automatically on the classpath).  Note that we don't want to use $config_dir, as that is the JBoss config dir.  We'll probably need another parameter on the class to accept this, or maybe try to directly reference 
lightblue::eap::module::datasources::directory (not 100% sure that would work).